src/main/java/frc/robot/Constants.java

```diff
@@ -43,5 +43,10 @@ public final class Constants {
     public static final int RightHoldingMotorID = 4;
     public static final double IOMotorRPM = 0;
     public static final double HoldingMorerPower = 0;
+
+    // Multiply raw encoder output by this to convert that to feet travelled
+    public static final double kEncoderTicksToFeet = 0; //TBD
+    // For trajectory differentialDriveKinematics
+    public static final double distBetweenWheelsMeters = 0; //TBD
```

Comment:
I am not super familiar with it, but are you planning on using WPILib's sysid system for getting these values?

Reply:
Was planning on measuring them physically like in the past, I've never heard of sysid
Do you recommend trying that?